### DISCUSSION TITLE
Toggle icons

### DISCUSSION THREAD

**Discussion by destiper** [StreamController/StreamController#114] [Ideas]
Would be cool to be able to add toggles, in the official software I would use hotkeys on toggle mode so that I can represent an on/off state for things like mute and deafen. I'd like to be able to contribute although my python is very rusty

**Comment by Core447** [comment]
I'm currently working on states, that you can use to create toggles, but also more complex stuff. It's already available in the main branch if you want to test it (the main branch is not very stable at the moment, but should be usable).

**Reply by v4u6h4n** [reply]
Saw that the other day! Very cool :-) But switching to main from flatpak broke everything so didn't have a chance to test

**Reply by Core447** [reply]
Yeah, I need to write an automatic migration code for the final release
